pages/league.tsx

```diff
@@ -54,18 +58,19 @@ const League: NextPage = () => {
     level: 3,
     classes: ["CSE 123"],
   };
```

Comment:
Also, lets remove the testStaff variable from this list of dummy data. Since `testClass` uses it, replace the `teachers: [testStaff]` with `teachers; []`